proxy/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "fsxa-proxy-api",
+  "version": "10.20.1-alpha.41",
+  "description": "The JavaScript Content API Library a.k.a Content API is an interface handling data coming from the FirstSpirit CaaS and the Navigation Service.",
```

Comment:
the description should contain sth. like: "This package represents the proxy version of the content api, which runs on client side."